File: helpers/input.rb
def get_lines(script_path)
  script_number = File.basename(script_path, '.rb')

  puts "##########################"
  puts
  puts "#############{script_number}############"
  puts
  puts "##########################"
  puts

  File.readlines("#{File.dirname(script_path)}/../inputs/#{script_number}.txt")
end

File: helpers/input.py
class Input:
    def get_lines(script_path):

        script_number = script_path.split(".")[0]

        print("##########################")
        print
        print(f"############{script_number}############")
        print
        print("##########################")
        print

        with open(f"inputs/{script_number}.txt") as f:
            return f.readlines()
